src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <main>
+      <form method="post" action="http://localhost:8080/">
+        <fieldset class="fieldset">
+          <legend>Personal info:</legend>
+          <label class="form-field"> 
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+              required>
+          </label>
+  
+          <label class="form-field"> 
+            Name:
+            <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required>
+          </label>
+          
+          <label class="form-field"> 
+            How old are you:
+            <input 
+              type="text"
+              name="age"
+              value="12"
+              min="1"
+              max="100"
+              autocomplete="off"
+              checked
+              required>
+          </label>
```

Comment:
```suggestion
            <input 
              type="text"
              name="age"
              value="12"
              min="1"
              max="100"
              autocomplete="off"
              checked
              required
            >
          </label>
```
check the whole document